tests/tools/onert_train/src/onert_train.cc

```diff
@@ -223,50 +224,109 @@ int main(const int argc, char **argv)
       measure.set(num_epoch, num_step);
       for (uint32_t epoch = 0; epoch < num_epoch; ++epoch)
       {
-        std::fill(losses.begin(), losses.end(), 0);
-        for (uint32_t n = 0; n < num_step; ++n)
+        //
+        // TRAINING
+        //
         {
-          // get batchsize data
-          if (!generator(n, input_data, expected_data))
-            break;
-
-          // prepare input
-          for (uint32_t i = 0; i < num_inputs; ++i)
+          std::fill(losses.begin(), losses.end(), 0);
```

Comment:
The training codes are just modified their indentation to fit the same format in the validation codes.